src/tallies/tally_scoring.cpp

```diff
@@ -248,15 +248,18 @@ double get_nuclide_neutron_heating(const Particle* p, const Nuclide& nuc,
 {
   size_t mt = nuc.reaction_index_[rxn_index];
   if (mt == C_NONE) return 0.0;
+
   auto i_temp = p->neutron_xs_[i_nuclide].index_temp;
   if (i_temp < 0) return 0.0; // Can be true due to multipole
+
   const auto& rxn {*nuc.reactions_[mt]};
   const auto& xs {rxn.xs_[i_temp]};
   auto i_grid = p->neutron_xs_[i_nuclide].index_grid;
   if (i_grid < xs.threshold) return 0.0;
+
   auto f = p->neutron_xs_[i_nuclide].interp_factor;
   return (1.0 - f) * xs.value[i_grid-xs.threshold]
-    + f * xs.value[i_grid-xs.threshold+1];
+   + f * xs.value[i_grid-xs.threshold+1];
```

Comment:
Not sure I understand this one -- can you explain why a one-space indent is desired?

Reply:
Digging into this it was included in set of proposals as part of a diff and classified by PullRequest under "Spacing." There's no description, though, so it may have been the work of an IDE. I went ahead and reverted since I can't find an explicit benefit.
******
<p align="left">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>